Objects/funcobject.c

```diff
@@ -423,6 +423,29 @@ func_get_annotations(PyFunctionObject *op, void *Py_UNUSED(ignored))
         op->func_annotations = PyDict_New();
         if (op->func_annotations == NULL)
             return NULL;
+
+    }
+    if (PyTuple_CheckExact(op->func_annotations)){
```

Comment:
```suggestion
    if (PyTuple_CheckExact(op->func_annotations)) {
```

Put space after `)`.

Reply:
And you can use shorter variable name for local variables.